scripts/validate_map.py

```diff
@@ -108,7 +108,7 @@ def check_code_name_absent(
         synapse_id = row[0]
         dataset = row[1]
 
-        data = pd.read_csv(syn.get(synapse_id)["path"], low_memory=False)
+        data = pd.read_csv(syn.get(synapse_id)["path"], low_memory=False, encoding='ISO-8859-1')
```

Comment:
@Chelsea-Na I think we should push the responsibility of providing a file that adheres to UTF-8 standards. The reason for this is that downstream processing will have to use this files and that code may fail.

Reply:
@Chelsea-Na Actually on second thought - lets have a discussion about whether or not we want to support non UTF-8 encoding because I have encountered this in the cbioportal export code

Reply:
Stats Team said they will ensure the files are utf-8 encoded. I will close this PR.